Task6/Task6.java

```diff
@@ -0,0 +1,37 @@
+import java.util.Scanner;
+
+public class Task6 {
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.print("Enter a four-digit natural number (between 1000 and 9999): ");
+        int number = scanner.nextInt();
+
+        if (number < 1000 || number > 9999) {
+            System.out.println("Invalid value. Please enter a four-digit natural number within [1000 - 9999].");
+        } else {
+            int first2Digits = number / 100;
```

Comment:
You need to slip the numbe rinto 4 digits and to create a new 2 numbers from them:
 1st digit & 4th digit
2nd digit & 3rd digit

Input:
8435 from this create 85 and 43 

Output:
85 > 43